readthedocs/settings/base.py

```diff
@@ -539,6 +539,12 @@ def USE_PROMOS(self):  # noqa
                 'filename': os.path.join(LOGS_ROOT, 'debug.log'),
                 'formatter': 'default',
             },
+            'db': {
+                'level': 'DEBUG',
```

Comment:
We have to be pretty careful that this won't happen in production. That could end up being a large file.

Reply:
Maybe these settings can go into the `dev.py` file.